java/osgi/OsgiWrapper.java

```diff
@@ -56,9 +61,9 @@ public final class OsgiWrapper implements Callable<Integer> {
   private String classpath;
 
   @Option(
-      names = {"--automatic_module_name"},
-      description = "The automatic module name of the bundle")
-  private String automaticModuleName;
+      names = {"--module_name"},
+      description = "The JPMS module name of the bundle")
+  private String moduleName;
```

Comment:
The OSGi Wrapper needs some updates; this updates the `--automatic_module_name` flag to just `--module_name`. While OSGi does not actually need to be aware of the module name (the build does not use BND tools to generate the `module-info.java`), it's there already, so why not; maybe it can be useful for later validation steps.

Reply:
May as well remove it? I only added it in https://github.com/protocolbuffers/protobuf/pull/14562 as a stop-gap measure until the real work to add a module-info.java file (that is, this mr) was made.

Reply:
@Sineaggi Roger that